app/controllers/admin/volunteers.py

```diff
@@ -111,16 +112,16 @@ def volunteerDetailsPage(eventID):
     if not (g.current_user.isCeltsAdmin or (g.current_user.isCeltsStudentStaff and g.current_user.isProgramManagerForEvent(event))):
         abort(403)
 
-    eventRsvpData = list(EventRsvp.select().where(EventRsvp.event==event))
-    eventParticipantData = list(EventParticipant.select().where(EventParticipant.event==event))
+    eventRsvpData = list(EventRsvp.select(EmergencyContact, EventRsvp).where(EventRsvp.event==event).join(EmergencyContact, JOIN.LEFT_OUTER, on=(EmergencyContact.user==EventRsvp.user)))
+    eventParticipantData = list(EventParticipant.select(EmergencyContact, EventParticipant).where(EventParticipant.event==event).join(EmergencyContact, JOIN.LEFT_OUTER, on=(EmergencyContact.user==EventParticipant.user)))
```

Comment:
Your `.join()` should come before your `.where()`

Reply:
Changed the order of the query methods.